_pages/en/software-assessment-sharing-pilot.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Software Assessment Sharing Pilot
+ref: software-assessment-sharing-pilot
+lang: en
+status: In Progress;primary
+sections: Pilots
+permalink: /software-assessment-sharing-pilot.html
+---
+
+- [Problem Statement](#problem-statement)
+- [Expected Results](#expected-results)
+- [GCxchange Group](#gcxchange-group)
+- [Sharing ESDC Software Assessments](#sharing-esdc-software-assessments)
+- [Feedback](#feedback)
+- [Next Steps](#next-steps)
+
+The idea of sharing software assessments documents and results between departments has been discussed many times on GCconnex over the years but was never formally put in place.
+Some files were shared but they are not easy to find or were only shared through email.
+GCxchange is a new platform available to all GC departments and agencies, that have onboarded to M365.
```

Comment:
You could summarize the next four lines into one. Something like:
"Through a one year pilot, The IT Strategy team will leverage the new GCExchange platform as a repository for software assessments so that they can be shared across the GC (confirmed to be one legal entity and so allowed by policy)"